packages/fluentui/react-northstar/src/utils/positioner/types.ts

```diff
@@ -121,6 +121,13 @@ export interface PositioningProps {
    * `position` props, regardless of the size of the component, the reference element or the viewport.
    */
   unstable_pinned?: boolean;
+
+  /**
+   * Change the size of the popper to fit it within the available space in viewport.
+   * true enables this for both width and height.
+   * 'height' enables this only for height and 'width' enables this only for width
```

Comment:
I suggest to update there description to clarify that it applies only `maxWidth` and/or `maxHeight` as there is no guarantee that it will work in the same way for other components (`Menu`, `Toolbar`) like for `Popup` that has tweaked styles for `overflow`.

Reply:
Nice suggestion. I changed it to be:
```ts
  /**
   * Applies max-height and max-width on popper to fit it within the available space in viewport.
   * true enables this for both width and height.
   * 'height' applies only `max-height` and 'width' for `max-width`
   */
  autoSize?: 'height' | 'width' | boolean;
```